gradle.properties

```diff
@@ -20,4 +20,7 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
```

Comment:
This can be used, however it's going to be tricky. I would suggest putting this in a constants class

```
class Constants {
  companion object {
    const val API_KEY = "4efe3434-b613-4733-a4de-8d3e6268365a"
   }
}